elyra/templates/components/generic_properties_template.jinja2

```diff
@@ -189,6 +193,22 @@
             "keyValueEntries": true
           }
         },
+        {
+          "parameter_ref": "elyra_kubernetes_secrets",
+          "control": "custom",
+          "custom_control_id": "StringArrayControl",
+          "label": {
+            "default": "Kubernetes Secrets"
+          },
+          "description": {
+            "default": "Kubernetes secrets to make available as environment variables to this node. The secret name and key given must be present in the Kubernetes namespace where the node is executed or this node will not run.",
+            "placement": "on_panel"
+          },
+          "data": {
+            "placeholder": "env_var_name=secret_name:secret_key",
```

Comment:
To be consistent with the input format for environment variables, we should use `env_var` instead of `env_var_name` now that we won't have a dedicated input widget.

```suggestion
            "placeholder": "env_var=secret_name:secret_key",
```